src/viur/core/modules/file.py

```diff
@@ -869,6 +944,106 @@ def download(self, blobKey: str, fileName: str = "", download: bool = False, sig
 
         raise errors.Redirect(signedUrl)
 
+    @exposed
+    def serve(
```

Comment:
How can I generate these URLs? How do I know the _host prefix_?

Reply:
the serving url must be split. The only alternative could be to save the split in the skel.

for js frontends there will be a function in vue-utils. 
for jinja we could add a filter for this, if needed

Reply:
Okay, Can you extend `File.create_download_url()` or implement a similar method for this?

Reply:
There are now 2 new methods:
create_serve_parameters: takes a serving url and splits to serve parameters host and key
create_serve_url function: generates serve url from a google servingurl, structure will be like this:
/file/serve/lh3/SOMEKEY/2000/test.webp?options=r270-fv&download=true